CMSgov/mint-app
# [EASI-2854] - Part 3 - Payments & Collaborators Translation Restructure

# EASI-2854

## Changes and Description

- Translation rework for Payments
- Translation rework for updating status form
- Translation rework of collaborators/new plan
- Deprecated lots of unneeded translation utils and files
- Removed unneeded field on `operationalNeedMap`
- Updated unit test
- Updated snapshots for a few small corrections and updated mock data



## How to test this change

Nothing should have changed visually, unless there was an incorrect render/text in the first place

- Verify MINT text/render matches Figma

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
